aws/structure.go

```diff
@@ -113,6 +113,32 @@ func expandEcsVolumes(configured []interface{}) ([]*ecs.Volume, error) {
 			}
 		}
 
+		configList, ok := data["docker_volume_configuration"].([]interface{})
+		if ok && len(configList) > 0 {
+			config := configList[0].(map[string]interface{})
```

Comment:
We may also want to add a `nil` check for `configList[0]` for this at some point in the future to prevent a potential panic, since things can get a little weird if the nested configuration has all zero-values -- see also: #5852